### PR TITLE
Add instagram story

### PR DESCRIPTION
**Pull request by sanghosuh**
Please review this, @casey-collab

### CODE REVIEW

File: korean/instagram/insta.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+인스타그램에 인증샷을 올린다. 하트 숫자가 늘어나는 것을 보고 기분이 좋아졌다.
+
+이제 어떻게 할까요?
+
+[벽을 먹는다](eating-walls/eating-marshmallows.md)?
```
Comment (casey-collab) [inline korean/instagram/insta.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: korean/instagram/insta.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[다시 잔다](sleep/marshmallow.md)?
```
Comment (casey-collab) [inline korean/instagram/insta.md]: This link is also invalid.

File: korean/instagram/insta.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+[노래를 부른다](sing-a-song/song.md)?
```
Comment (casey-collab) [inline korean/instagram/insta.md]: This link is also invalid.

File: korean/instagram/insta.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+[마쉬멜로라면 핫초코지~! 핫초코를 사러 나간다.](coco/coco.md)
```
Comment (casey-collab) [inline korean/instagram/insta.md]: This link is also invalid.

File: korean/instagram/insta.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+[카페 성수에 간다](cafe-sungsu/cafe-sungsu.md)
```
Comment (casey-collab) [inline korean/instagram/insta.md]: This link is also invalid.

File: korean/instagram/insta.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+인스타그램에 인증샷을 올린다. 하트 숫자가 늘어나는 것을 보고 기분이 좋아졌다.
```
Comment (casey-collab) [inline korean/instagram/insta.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sanghosuh** [comment]
Fixed broken links and added a link to my page on marshmallow.md, thereby fixing all the errors pointed out by @casey-collab

File: korean/instagram/insta.md (hunk 6 above)
Comment (casey-collab) [inline korean/instagram/insta.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sanghosuh** [comment]
Make my page reachable from language.md. Please take a look, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**